plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -18,7 +18,7 @@ const { expectRevert, constants } = require('openzeppelin-test-helpers');
 
 const { VAULT_ID, TX_TYPE, SAFE_GAS_STIPEND } = require('../../../helpers/constants.js');
 
-contract('PaymentExitGame', () => {
+contract('PaymentExitGame', ([_, _maintainer, _authority, richFather]) => {
```

Comment:
I am _guessing_ you'd like to demonstrate the init function would fail if not called by the maintainer thus putting maintainer var name on the list here. However, this unfortunately is not how this test is currently setup. Different from how migration script is used, in the test here it takes the short cut of using `SpyPlasmaFrameworkForExitGame` which would just lazily put the deployer as maintainer and authority address too. The position of addresses here would give false illustration to the code reader.

https://github.com/omgnetwork/plasma-contracts/blob/c5002f3137f21114b8d5f834485e1b729ca09d8c/plasma_framework/contracts/mocks/exits/SpyPlasmaFrameworkForExitGame.sol#L23-L28

If you'd like to demonstrate this maintainer, you could probably tweet the mock contract construction to enforce it passes maintainer address. For other existing tests you could pass in the deployer (first address of the list) and pass in a different maintainer address in the test here.

Or.....just I think I am to just remove `_maintainer`, `_authority` here to not five false illustration. Probably left a comment on the how SpyFramework would setup the maintainer as deployer if want to make the test setup clearer as the short cut has caused some confusion.

Same comment apply for the other tests that you put `_maintainer` on the address lists.